README.md

```diff
@@ -30,7 +30,7 @@ GitHub's [Hello World]
 GitHub's [Open Source Guide]
 GitHub's [Video Training]
 
-[CONTRIBUTING]: https://github.com/LMHT/LMHT.github.io/blob/master/contributing.md  
+[CONTRIBUTING]: https://github.com/LMHT/LMHT.github.io/blob/master/CONTRIBUTING.md  
```

Comment:
I would not hard-code the repository that this link points to. If you instead do

```markdown
[CONTRIBUTING]: CONTRIBUTING.md
```

That should link to the file named 'CONTRIBUTING.md' in whichever repo you're currently it in, regardless of name, user, company, etc.